draft-ietf-httpbis-digest-headers.md

```diff
@@ -77,442 +76,413 @@ normative:
+of SEMANTICS}}). This supports more advanced use cases such as validating the
+integrity of a resource the was reconstructed from parts retrieved using
+multiple requests or connections.
+
+This document obsoletes RFC 3230 and therefore the Digest and Wants-Digest HTTP
+fields; see {{obsolete-3230}}.
+
 
 ## Document Structure
 
 This document is structured as follows:
 
-- {{digest}} describes concepts related to representation digests. It also
-  defines the Digest request and response header and trailer field,
+- {{representation-digest}} defines the Representation-Digest request and response header and trailer field,
 - {{content-digest}} defines the Content-Digest request and response header and trailer field,
-- {{want-fields}} defines the Want-Digest and Want-Content-Digest request and response header and
+- {{want-fields}} defines the Want-Representation-Digest and Want-Content-Digest request and response header and
   trailer field,
-- {{algorithms}} describes algorithms and their relation to Digest,
+- {{algorithms}} describes algorithms and their relation to the fields defined in this document,
 - {{state-changing-requests}} details computing representation digests,
 - {{examples-unsolicited}} and {{examples-solicited}} provide examples of using
-  Digest and Want-Digest.
+  Representation-Digest and Want-Representation-Digest.
 
 ## Concept Overview
 
-This document defines the `Digest` request and response header and trailer
-field; see {{digest}}. At a high level, the value contains a checksum, computed
-over `selected representation data` ({{Section 3.2 of
-SEMANTICS}}), that the recipient can use to
-validate integrity. Basing `Digest` on the selected representation makes it
-straightforward to apply it to use-cases where the transferred data requires
-some sort of manipulation to be considered a representation or conveys a
-partial representation of a resource, such as Range Requests (see {{Section
-14.2 of SEMANTICS}}).
-
-To support use-cases where a simple checksum of the content bytes is required,
-this document introduces the `Content-Digest` request and response header and trailer field; see
+The HTTP fields defined in this document can be used for HTTP integrity. Senders
+choose a hashing algorithm and calculate a checksum from an input related to the
+HTTP message, the algorithm identifier and checksum are transmitted in an HTTP
+field. Receivers can validate the checksum for integrity purposes. Hashing
+algorithms are registered in the HTTP Digest Algorithm Values Registry (see
+{{algorithms}}).
+
+Selecting the data on which checksums are calculated depends on the use case of
+HTTP messages. This document provides different headers for HTTP representation
+data and HTTP content.
+
+This document defines the `Representation-Digest` request and response header
+and trailer field ({{representation-digest}}) that contains a checksum value
```

Comment:
```suggestion
This document defines the `Representation-Digest` field ({{representation-digest}}) that contains a value
```

Reply:
That's a decent suggestion, let's track it in https://github.com/httpwg/http-extensions/issues/1941